.github/workflows/deploy-docker.yml

```diff
@@ -0,0 +1,34 @@
+name:                           Docker Image Release
+
+on:
+  push:
+    branches:
+      - stable
+    tags:
+      - v2*
+
+jobs:
+  deploy-docker:
+    name:                       Build Release
+    runs-on:                    ubuntu-latest
+    steps:
+      - name:                   Checkout sources
+        uses:                   actions/checkout@master
+      - name:                   Install toolchain
+        uses:                   actions-rs/toolchain@v1
+        with:
+          toolchain:            stable
+          profile:              minimal
+          override:             true
+      - name:                   Deploy to docker hub
+        uses: elgohr/Publish-Docker-Github-Action@master
+        env:
+          VCS_REF: $GITHUB_SHA
+          BUILD_DATE: $(date -u '+%Y-%m-%dT%H:%M:%SZ')
+        with:
+          name: openethereum/openethereum
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          dockerfile: scripts/docker/alpine/Dockerfile
+          buildargs: VCS_REF,BUILD_DATE
```

Comment:
where are these args used?

Reply:
copied it from this https://github.com/openethereum/openethereum/blob/10d82ef119468f04284addb050f147755df8cbc9/scripts/docker/hub/publish-docker.sh#L21

Wasn't sure if it could be removed it

Reply:
ok, yeah, is not included in the alpine image like it was in the previous
https://github.com/openethereum/openethereum/blob/10d82ef119468f04284addb050f147755df8cbc9/scripts/docker/hub/Dockerfile#L4